src/Algebra/WithApartness/Bundles.agda

```diff
@@ -0,0 +1,54 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Bundles for local algebraic structures
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.WithApartness.Bundles where
+
+open import Level using (_⊔_; suc)
+open import Algebra.Core using (Op₁; Op₂)
+open import Algebra.Bundles using (CommutativeRing)
+open import Algebra.WithApartness.Structures using (IsHeytingCommutativeRing; IsHeytingField)
+open import Relation.Binary.Core using (Rel)
+open import Relation.Binary.Bundles using (ApartnessRelation)
+
+record HeytingCommutativeRing c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) where
+  field
+    Carrier                  : Set c
+    _≈_                      : Rel Carrier ℓ₁
```

Comment:
Again, we need fixity declarations for all of these (see `Algebra.Bundles` for examples)

Reply:
Thanks. I've taken all the algebraic operation fixities from CommutativeRing.